sdk/tests/route-facts.test.ts

```
@@ -0,0 +1,575 @@
+    ]);
+    expect(row.routes.tags.noToolCalled).toMatchObject({
+      state: "measured",
+      numerator: 1,
+      denominator: 2,
+      value: 0.5,
+    });
+  });
+
+  test("exclusions by status and evaluator error leave the denominator", () => {
+    const doc = build([
+      trial({ trialKey: "ok", result: "passed", actualToolCalls: [call("tool_a")] }),
```

ESLint

[error] Replace `·trialKey:·"ok",·result:·"passed",·actualToolCalls:·[call("tool_a")]` with `⏎········trialKey:·"ok",⏎········result:·"passed",⏎········actualToolCalls:·[call("tool_a")],⏎·····`

(prettier/prettier)